### PR TITLE
add 6/7 Seattle incident where woman stops breathing

### PR DESCRIPTION
**Pull request by sprucelee**
This is a re-submit of #448 which was closed b/c it lacked video evidence.

This PR includes a video on twitter showing the protester directly hit by projectile, falling to the ground and becoming unresponsive. Others drag them away from the police. The details of what happens after the video ends are in the firsthand account from a medic on reddit.

### CODE REVIEW

**Comment by rsarvar1a** [comment]
@emarcey I think the video + account is enough, could I get a second opinion though?

File: reports/Washington.md [hunk 1]
```diff
@@ -149,6 +149,15 @@ id: wa-seattle-12
 * https://www.reddit.com/r/2020PoliceBrutality/comments/gycscp/cant_go_1_day_without_teargaslighting_us/?utm_source=share&utm_medium=web2x
 * https://www.forbes.com/sites/jemimamcevoy/2020/06/08/seattle-police-use-tear-gas-against-protestors-despite-city-ban/#7e98a1d5b4bc
 
+### Woman hit with police projectile resumes breathing after CPR, delivered to hospital | June 7th
+
+Medics attempt to treat "a young woman in her 20s who had taken a police projectile to her chest" as police throw flash bang grenades directly into the first aid center. There were several cycles of the victim no longer responding or breathing but becoming responsive again after CPR. 911 could not send an ambulance. Eventually a civilian transported her by car to the ED where she was delivered with a pulse.
```
Comment (emarcey) [inline reports/Washington.md:154]: Am I missing something related to hospitals or should `ED` be `ER`

Comment (sprucelee) [reply reports/Washington.md:154]: Some ERs are apparently called Emergency Departments. The firsthand account used ED so I followed that.

**Comment by emarcey** [comment]
This looks good. I think a valuable additional piece of evidence is that the filmer reports the tear gas is in sufficient quantity to disturb the breathing of people in her office building unrelated to protests.

**Comment by sprucelee** [comment]
I've added the detail about people in the office building having trouble breathing. Also noticed that the filmer (who is male identified, btw) is an editor at the local alt weekly, which lends credibility to the account. I have mentioned that as well.

**Comment by emarcey** [comment]
Ah thanks for the correction. Looks good to merge to me.

**Comment by sprucelee** [comment]
thank you all so much for your work!

**Pull request merged**